clients/POSTGRESQL/pg.js

```diff
@@ -0,0 +1,13 @@
+// Binding for kafkajs client
```

Comment:
I assume this should say `postgres client` in the comment?

Reply:
Thanks for catching that.